docs/how_to/create_custom_map.rst

```diff
@@ -59,7 +60,7 @@ Here's an example of creating a header from some generic data and an `astropy.co
     rsun_obs: 965.3829548285768
 
 From this we can see now that the function returned a `sunpy.util.MetaDict` that populated the standard FITS keywords with information provided by the passed `astropy.coordinates.SkyCoord`, and the data array.
-Since the ``reference_pixel`` and ``scale`` keywords were not passed in the example above, the values of the ``crpix`` and ``cdelts`` keys were set to the center of the data array and 1, respectively, by default.
+Since the ``reference_pixel`` and ``scale`` keywords were not passed in the example above, the values of the "crpixi" and "cdelti" keys were set to the center of the data array and 1, respectively, by default.
```

Comment:
This reads pants, I think this might be better:

```suggestion
Since the ``reference_pixel`` and ``scale`` keywords were not passed in the example above, the values of the "CRPIXn" and "CDELTn" keys were set to the center of the data array and 1, respectively, by default.
```